assemblies/assembly-bulk-importing-from-github.adoc

```diff
@@ -0,0 +1,18 @@
+[id="bulk-importing-github-repositories"]
+= Bulk importing GitHub repositories
+
+[IMPORTANT]
```

Comment:
@themr0c The technology preview note has been added as follows. I would suggest we keep it consistent.
![image](https://github.com/user-attachments/assets/356fa14b-6f97-437f-9567-e1d6a0239d96)